content/blog/api-console-for-data-services-cloud-console.md

```diff
@@ -0,0 +1,91 @@
+   5. URL to the HPE DSCC end-point (string)
+2. Generate Access Token, change Client Secret, and delete Access Token.
+
+![API Gateway](/img/api-gateway-block.png "DSCC API Gateway")
+
+### Manage API Client Applications
+
+Each of the API client Applications Connection creates the relationship between the client and server (application). Click on the **Create Credentials** button to generate a client credential to generate the Access Token.
+
+![](/img/create-credentials-button.png)
+
+Inside the Create Credentials menu, click on the V button to show the pull down list and use the mouse to click on the desired application. i.e. For storage select the Data Services Cloud Console. The list also show region where the applications is deployed.
+
+![select the application](/img/select-the-desired-application.png "Choose application (DSCC)")
+
+After selecting the correct application enter the Credential Name (Please see HPE GreenLake Cloud Console user guide for supported characters.) Click the Create Credentials button to proceed with Client Credentials creation.
+
+![Create Credential no input yet](/img/create-credentials-menu.png "Generate Client Credentials 1st time")
+
+Note that to create the credentials for the OAuth, you have to provide the application and the name of this credential. Select the Data Services Cloud Console Application on the appropriate region where the console are deployed for this organization.
+
+Once the Create Credential is selected, the following information about the OAuth will be shown.
+
+![](/img/api-client-credential-created.png "Credentials Created Close")
+
+After closing the credential creation menu, user can observe the created API Client Credentials
+
+![](/img/application-credential-created-prior-shown.png "API Client Credentials are created")
+
+Clicking on the down arrow, user can generate the access token required to use the API.
+
+![](/img/api-client-credential-get-access-token.png "Time to obtain the Access Token")
+
+After clicking on the Generate Access Tokens button, the menu requires user to enter the Client Secret obtained from the API Client Credentials 
+
+![](/img/generate-access-token-with-secret.png "Use the client secret to generate Access Token")
+
+Copy the Access Token for the API usage.
+
+![](/img/access-token-created-and-close.png "Access Token Generated and Consumed")
+
+User then can embed the access token to the REST API request in order to perform the HTTP method against the desired resource in order to obtain the response.  
+
+### Nice! Can you give me an example of using the access token?
+
+The recommended tool at this moment of time to experiment with the REST API for DSCC is the Postman which is downloadable from the [Postman website](https://www.postman.com/downloads/). The postman is a versatile tool, where we can copy the access token that we obtained and issue a REST API request in simple and format the response beautifully. 
```

Comment:
Should be "at this moment in time used to experiment"
Add a comma after "DSCC is the Postman"/prior to "which is downloadable"
If "Postman" is capitalized in the first sentence, it should also be in the second sentence "The Postman is a versatile tool...

I don't really understand the last part of the last sentence, though. Let me see if I can decode:
"The Postman is a versatile tool. It can be used to copy the access token that we obtained and issue a REST API request in a simple format that responds beautifully."??